config/index.js

```diff
@@ -16,21 +16,21 @@ const sentryDsn = 'https://1f74b885d0c44549b57f307733d60351:dd736ff3ac994104ab66
 const DEFAULT_TIMEOUT = 2000;
 
 const overridableFields = [
-	'tokenKey', 'tokenPassword', 'testPackId', 'concurrency', // launcher automated
+	'tokenKey', 'tokenPassword', 'token', 'testPackId', 'concurrency', 'presets', // launcher automated
 	'username', 'password', 'orgId', 'deviceId', 'appConfigId', 'inspect', 'inspectBrk', // launcher intaractive
 	'logLevel', 'logDir', 'timestamp', 'configFile', 'disallowCrashReports', 'defaultTimeout', // launcher common
 ];
 
 const configurableFields = ['logLevel', 'disallowCrashReports', 'defaultTimeout'];
 
 const main = Object.freeze({
-	apiUrl: 'https://the.suite.st/api/public/v3',
+	apiUrl: 'https://integ2.suite.st/api/public/v4',
```

Comment:
pls revert testing code

Reply:
I recommend force push, testing code should not be in the code base even temporary